src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -210,9 +234,12 @@ export const restoreBrowserTrackStates: ActionCreator<ThunkAction<
     });
   });
 
-  browserMessagingService.send('bpane', {
-    off: tracksToTurnOff,
-    on: tracksToTurnOn
+  browserMessagingService.send({
+    action: ChromeToBrowserMessagingActions.TOGGLE_TRACKS,
+    payload: {
+      off: tracksToTurnOff,
+      on: tracksToTurnOn
+    }
```

Comment:
As a possible api:

```
browserMessagingService.send(toggleTracksMessage({
  off: tracksToTurnOff,
  on: tracksToTurnOn
});
```

where `toggleTracksMessage` would be an action creator that will know that it wants to add `{action: ChromeToBrowserMessagingActions.TOGGLE_TRACKS}` and put whatever came in the arguments in the `payload` field.

Wouldn't it be nicer?

Reply:
Sounds good! Do you think it will be better to define these within `browser-message-creator` itself?

Reply:
I noticed that most of the actions are used only once so I'm wondering if it would be simpler to change `browserMessagingService.send({action, payload})` to `browserMessagingService.send(action, payload)`?

Reply:
> Sounds good! Do you think it will be better to define these within browser-message-creator itself?

As opposed to defining them in redux- or react-related files? Yes, I think so.

> wondering if it would be simpler to change browserMessagingService.send({action, payload}) to browserMessagingService.send(action, payload)

With `browserMessagingService.send({action, payload})` you'll be able to simply pass to `browserMessagingService.send` the output of a browser-message-creator function. With `browserMessagingService.send(action, payload)` you'll need to unwrap that output into two variables and pass them individually. I think  `browserMessagingService.send({action, payload})` looks more ergonomic. The `.send` is acting almost like redux's `dispatch`, which takes an action object rather than individual components of that object.

Reply:
No, I was asking if we could just use `browserMessagingService.send(action, payload)` and avoid message-creator function. I am not sure if it is worth creating a message-creator function for each and every actions.

Reply:
I may not have the proper appreciation of which way is going to be more convenient; but from looking at this PR it feels that encapsulating all browser message-related stuff in their own file/files might be tidier than creating action objects from scratch in redux files on react components.

I may be wrong though. Would love others' feedback during code review.